Applications/web_portal/web_app/library/source/pages/dashboard_page/dashboard_page.tsx

```diff
@@ -11,8 +11,20 @@ interface Properties {
   /** The model to display. */
   model: DashboardModel;
 
-  /** The action to perform when logging out. */
-  onLogout?: () => void;
+  /** The action to perform when navigating to the profile. */
```

Comment:
The dashboard page internally handles all of these actions, so no prop should be exposed for these. The only action the dashboard page can not possibly handle is the log out, which is why that has its own prop.